src/main/java/at/hannibal2/skyhanni/features/bingo/BingoBoopParty.kt

```diff
@@ -0,0 +1,39 @@
+package at.hannibal2.skyhanni.features.bingo
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.hypixel.chat.event.PrivateMessageChatEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RegexUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.getPlayerNameFromChatMessage
+import at.hannibal2.skyhanni.utils.StringUtils.trimWhiteSpaceAndResets
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object BingoBoopParty {
+
+    val config get() = SkyHanniMod.feature.event.bingo.boopParty
+    val patternGroup = RepoPattern.group("bingo")
+    private val boopPattern by patternGroup.pattern(
+        "boop",
+        "§dFrom.*§d§lBoop!"
+    )
+
+    @SubscribeEvent
+    fun onChat(event: PrivateMessageChatEvent) {
+        if (!isEnabled()) return
+
+        val msg = event.message.trimWhiteSpaceAndResets()
+        boopPattern.matchMatcher(msg) {
```

Comment:
event.message is the stuff after From name: 
so this will never match.
Instead you could check if event.message is whatever it shows when its a boop and then use event.author